Mail SSL encryption was dropped (#1832)
PrestaShop/docs: modules/core-updates/9.0.md

- `symfony/uid`
- `symfony/workflow`

{{% notice note %}}
**Mail SSL encryption was dropped**

> When we were migrating Swift mailer to Symfony Mailer, we noticed that ssl support was not an option in ESMTP transport. Also SSL is an old and outdated encryption type. For security reasons, it will not be allowed anymore.

Reviewer: ```suggestion
When we migrated from Swift Mailer to Symfony Mailer, we noticed that SSL support was not an option in ESMTP transport. SSL is an old and outdated encryption type, and for security reasons, it will no longer be allowed.
```

> The remaining choices are tls encryption or no encryption.

Reviewer: ```suggestion
The remaining choices are "TLS encryption" or "No encryption".
```
